controller/ovn-controller.c

```diff
@@ -1154,9 +1182,13 @@ static bool
 runtime_data_ovs_interface_handler(struct engine_node *node, void *data)
 {
     struct ed_type_runtime_data *rt_data = data;
+    struct ed_type_runtime_tracked_data *tracked_data = node->tracked_data;
```

Comment:
call
 en_runtime_clear_tracked_data(node->tracked_data);

here as well

Reply:
No need. engine will take care of clearing the tracked data.